scripts/mixedos_sonobuoy.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Usage: sh mixedos_sonobouy.sh install 0.56.17 amd64
+# Usage: sh mixedos_sonobouy.sh delete
+
+action=$1
+
```

Comment:
Can we have variables for $2 and $3 as `version` and `arch` so that this shell file is more clear too and easier to make updates to?

Reply:
Updated